content/blogposts/2023/cody-vscode-0-12-release.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Cody for VS Code v0.12 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Tim Lucas
+    url: https://handbook.sourcegraph.com/team/#tim-lucas
+  - name: Beatrix Woo
+    url: https://handbook.sourcegraph.com/team/#beatrix-woo  
+publishDate: 2023-09-20T0:00-07:00
+description: "new onboarding & sign in experience, more custom configuration options, and improved Cody performance"
+tags: [blog]
+slug: "cody-vscode-0-12-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.12.0-og-image.jpg
+socialImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.12.0-og-image.jpg
+---
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v0.12 is now available and includes a new onboarding & sign in experience, more custom configuration options, and improved Cody performance.
+
+### User Onboarding and Sign-in Experience
+
+New users can now sign up and sign in for Cody using their Github and Gitlab accounts. 
+
+### Cody Pre-Instructions
+
+You can now add a custom starter message that prepend chat requests to Cody by setting the `cody.chat.preInstruction` configuration in `settings.json`.
+
+```json
+{
+    "cody.chat.preInstruction": "My name is Jeff."
```

Comment:
maybe a better example lol?

Reply:
```suggestion
    "cody.chat.preInstruction": "I am a frontend engineer."
```

Reply:
i took it from you! 😆 

let me add to it